core/src/main/java/io/jenkins/pluginhealth/scoring/probes/ContinuousDeliveryProbe.java

```diff
@@ -24,72 +24,18 @@
 
 package io.jenkins.pluginhealth.scoring.probes;
 
-import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.util.Map;
-import java.util.Objects;
-import java.util.stream.Stream;
-
-import io.jenkins.pluginhealth.scoring.model.Plugin;
-import io.jenkins.pluginhealth.scoring.model.ProbeResult;
-
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.springframework.core.annotation.Order;
 import org.springframework.stereotype.Component;
 
+/**
+ * This probe checks if continuous delivery (CD) workflow is configured in the GitHub Actions
+ */
 @Component
 @Order(ContinuousDeliveryProbe.ORDER)
-public class ContinuousDeliveryProbe extends Probe {
-    private static final Logger LOGGER = LoggerFactory.getLogger(ContinuousDeliveryProbe.class);
-    public static final int ORDER = LastCommitDateProbe.ORDER + 100;
+public class ContinuousDeliveryProbe extends AbstractGitHubWorkflowProbe {
+    public static final int ORDER = AbstractGitHubWorkflowProbe.ORDER + 20;
```

Comment:
As the two probes are not dependent to each others, we can use the same order here.

```suggestion
    public static final int ORDER = AbstractGitHubWorkflowProbe.ORDER;
```